examples/demo-apps/apple_ios/default-ios-device-farm-appium-test-spec.yml

```diff
@@ -0,0 +1,31 @@
+version: 0.1
+
+# Phases are collection of commands that get executed on Device Farm.
+phases:
+  # The install phase includes commands that install dependencies that your tests use.
+  # Default dependencies for testing frameworks supported on Device Farm are already installed.
+  install:
+    commands:
+
+  # The pre-test phase includes commands that setup your test environment.
+  pre_test:
+    commands:
+      - mkdir $DEVICEFARM_TEST_PACKAGE_PATH/Debug-iphoneos
+      - unzip $DEVICEFARM_APP_PATH -d /tmp
+      - mv /tmp/Payload/*.app $DEVICEFARM_TEST_PACKAGE_PATH/Debug-iphoneos/
+
+  # The test phase includes commands that run your test suite execution.
+  test:
+    commands:
+      - xcodebuild test-without-building -destination id=$DEVICEFARM_DEVICE_UDID -xctestrun $DEVICEFARM_TEST_PACKAGE_PATH/*.xctestrun -derivedDataPath $DEVICEFARM_LOG_DIR
```

Comment:
IIUC this is unlike android the model is bundled to the test already, so running the test would be running the model

Reply:
Yup, the model is already included, so there is no need to upload it separately like the extra data archive like Android test